src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,34 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+
+  health = 100;
+
+  constructor(name) {
+    this.name = name;
+    Animal.alive.push(this);
+  }
 }
 
 class Herbivore extends Animal {
-  // write your code here
+  hidden = false;
```

Comment:
The hidden property is well-handled, but consider explicitly setting it in the constructor to avoid confusion about which animals have it
```suggestion
   constructor(name) {
    super(name);
    this.hidden = false;
  }
```